### ISSUE TITLE
Sometimes messages.chat() not present.

### ISSUE THREAD

**Issue by UnamedRus** [Lonami/grammers#45]
```
thread 'main' panicked at 'called `Option::unwrap()` on a `None` value', /home//.cargo/git/checkouts/grammers-a13650801c60c1ef/1c79a48/lib/grammers-client/src/types/message.rs:177:14
```

https://github.com/Lonami/grammers/blob/802f3829db624ef1ef6c689eb9af5dffe5e97b9d/lib/grammers-client/src/types/message.rs#L177

**Comment by Lonami** [comment]
Telegram does not always send chat information, sometimes only the peer type and ID. However the library should use `min` constructors where appropriated to make sure there is *always* an usable chat.

**Comment by Xobtah** [comment]
I'm having the same problem

**Comment by Hartbook** [comment]
Hello,
Is there really no way to retrieve the chat (I only need the id) where the message has been sent ?
The only way I can think of is iterating over all messages of all dialogs, to find a match, but that would be terribly inefficient.

**Comment by Lonami** [comment]
I've released 0.4.0 with the above fix. Sorry for the long wait and inconvenience. (I would still recommend to use the git version in general since that's likely to contain the latest fixes, and CI ensures it builds fine).
